libass/ass_drawing.c

```diff
@@ -257,8 +257,8 @@ bool ass_drawing_parse(ASS_Outline *outline, ASS_Rect *cbox,
 
     if (lib)
         ass_msg(lib, MSGL_V,
-                "Parsed drawing with %d points and %d segments",
-                outline->n_points, outline->n_segments);
+                "Parsed drawing with %llu points and %llu segments",
```

Comment:
`%zu` doesn't work on some MSVCRT versions, and I wasn't sure if we wanted to support them or not.

Reply:
Both `ll` and `z` introduced in C99 and it's libass requirement anyway. Also a quick search shows that MSVC supports `z` at least since 2015. I think it's safe to use.

Reply:
I've run into issues with this in MSVCRT in the past (where lld worked fine), but we do already have a `%zu` usage in `decode_font`, so sure, fair enough.